contents/docs/libraries/js/index.mdx

```diff
@@ -580,6 +580,19 @@ posthog.init('my_token', {
 })
 ```
 
+### Or send no events while developing
+
+When working locally it can be useful to see what posthog would do, without actually sending the data to PostHog
+
+```
+posthog.init('my_token', {
+    before_send: (event: CaptureResult): CaptureResult | null {
```

Comment:
this would give an type error because the event argument should be type 'CaptureResult | null':

`before_send: (event: CaptureResult | null): CaptureResult | null {`

so I think this should be updated in the docs